sct-commons/src/main/java/org/lfenergy/compas/sct/commons/ExtRefEditorService.java

```diff
@@ -399,6 +349,65 @@ private String computeDaiValue(String lnPrefix, TExtRef extRef, String daName) {
         }
     }
 
+   private void updateLDEPFDos(TDataTypeTemplates dtt, TIED tied, TLDevice tlDevice, TExtRef tExtRef, TChannel setting) {
+       // Digital
+        if (setting.getChannelType().equals(TChannelType.DIGITAL)) {
+            lnEditor.findLn(tlDevice, tAnyLN -> LnId.from(tAnyLN).lnClass().equals(LN_RBDR) && LnId.from(tAnyLN).lnInst().equals(setting.getChannelNum()))
+                    .ifPresent(tln -> updateDaiValue(dtt, tied, tlDevice, tln, tExtRef, setting));
+            lnEditor.findLn(tlDevice, tAnyLN -> LnId.from(tAnyLN).lnClass().equals(LN_RBDR) && LnId.from(tAnyLN).lnInst().equals(setting.getChannelNum())
+                            && LnId.from(tAnyLN).prefix().equals(LN_PREFIX_B))
```

Comment:
You chould put `LnService.matchesLn` in `LnEditor` interface and use it instead : 

```java
lnEditor.findLn(tlDevice, tAnyLN -> lnEditor.matchesLn(tAnyLN, LN_RBDR, setting.getChannelNum(), null))
```
```java
lnEditor.findLn(tlDevice, tAnyLN -> lnEditor.matchesLn(tAnyLN, LN_RBDR, setting.getChannelNum(), LN_PREFIX_B))
```